File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AbyssalCutlass.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword;

/* JADX INFO: loaded from: classes3.dex */
public class AbyssalCutlass extends Sword {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_sword_abyssal_cutlass_name;
        this.idDescription = R.string.weapon_sword_abyssal_cutlass_description;
        this.idImage = R.drawable.abyssal_cutlass;
        this.price = 5501L;
        this.constitution = 24;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AbyssalCutlass.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword

class AbyssalCutlass : Sword() {
    override fun configureProperties() {
        idName = R.string.weapon_sword_abyssal_cutlass_name
        idDescription = R.string.weapon_sword_abyssal_cutlass_description
        idImage = R.drawable.abyssal_cutlass
        price = 5501L
        constitution = 24
    }
}
